### DISCUSSION TITLE
Multi Block Build and Plugin/SlotFill (or Custom Data Store) not specifically related to a block (or specific block)

### DISCUSSION THREAD

**Discussion by aaronware** [WordPress/block-development-examples#85] [Examples Ideas]
There are multiblock examples and NO BLOCK examples already however not both together.

If you are using a Slotfill unrelated to a specific block, you'll need add a custom webpack.config.js to your src in order for the additional js to be taken into account. The config has to be very particular from what I have seen. Or else the build process will only emit [copied] the block.json in the build process an the other elements such as the js,assets.php will not be built.

**Comment by juanmaguitar** [comment]
Hi @aaronware! 

This seems like an interesting use case to add to this repo. 
Would you be up to drafting a simple use case for this example? 
You can add a PR directly on this repo by following [these conventions](https://github.com/WordPress/block-development-examples/wiki/06-Conventions-for-examples) or pass a link to any repo with the example (from there, I can help to migrating that example to this repo)
